lib/ocpp/v201/smart_charging.cpp

```diff
@@ -73,6 +73,24 @@ ProfileValidationResultEnum SmartChargingHandler::validate_evse_exists(int32_t e
                                               : ProfileValidationResultEnum::Valid;
 }
 
+ProfileValidationResultEnum SmartChargingHandler::validate_charging_station_max_profile(const ChargingProfile& profile,
+                                                                                        EvseInterface& evse) const {
+    if (profile.chargingProfilePurpose != ChargingProfilePurposeEnum::ChargingStationMaxProfile) {
```

Comment:
I think part of the change that @christopher-davis-afs  did was to only pass in the `evse_id` when that was the only thing that is needed. Does that make sense to do here?

Reply:
Yes, it does.